2025-09-04 13:02:50,315 - INFO - User registered successfully: esh@example.com
2025-09-04 13:09:30,471 - INFO - User registered successfully: akhi@example.com
2025-09-04 13:12:56,437 - WARNING - Registration failed: User already exists - ram@example.com
2025-09-04 13:13:43,484 - INFO - User registered successfully: ram@example.com
2025-09-04 14:22:43,324 - INFO - User logged in successfully: john@example.com
2025-09-04 14:25:19,133 - WARNING - Login failed: Incorrect password - john@example.com
2025-09-04 14:26:45,204 - INFO - User logged in successfully: john@example.com
2025-09-04 15:09:57,103 - INFO - Team created successfully: Warriors
2025-09-04 15:10:59,336 - INFO - Team created successfully: Thunders
2025-09-04 15:12:00,889 - INFO - Team created successfully: Rangers
2025-09-04 15:16:13,308 - WARNING - Team creation failed: Rangers already exists
2025-09-04 15:29:44,571 - INFO - Fetched all teams
2025-09-04 15:31:13,063 - INFO - Fetched all teams
2025-09-04 15:33:03,901 - INFO - Fetched all teams
2025-09-04 15:34:32,286 - INFO - Fetched all teams
2025-09-04 15:47:03,556 - INFO - Fetched all teams
2025-09-04 15:48:06,012 - INFO - Fetched all teams
2025-09-04 15:48:13,080 - INFO - Fetched all teams
2025-09-04 15:48:18,012 - INFO - Fetched all teams
2025-09-04 15:48:21,371 - INFO - Fetched all teams
2025-09-04 15:49:34,637 - INFO - Fetched all teams
2025-09-04 15:49:38,476 - INFO - Fetched all teams
2025-09-04 17:19:31,050 - INFO - User logged in successfully: esh@example.com
2025-09-04 17:26:18,124 - INFO - User logged in successfully: esh@example.com
2025-09-04 17:45:22,550 - INFO - User logged in successfully: esh@example.com
2025-09-04 17:46:48,956 - INFO - Team created successfully: Challengers
2025-09-04 17:48:49,567 - INFO - Team created successfully: Royel Challengers
2025-09-04 17:49:40,016 - ERROR - Invalid or expired token
Traceback (most recent call last):
  File "C:\Users\K K\anaconda3\envs\gameplan\Lib\site-packages\jose\jws.py", line 189, in _load
    header = json.loads(header_data.decode("utf-8"))
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x88 in position 6: invalid start byte

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\K K\anaconda3\envs\gameplan\Lib\site-packages\jose\jwt.py", line 159, in decode
    payload = jws.verify(token, key, algorithms, verify=verify_signature)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\K K\anaconda3\envs\gameplan\Lib\site-packages\jose\jws.py", line 74, in verify
    header, payload, signing_input, signature = _load(token)
                                                ^^^^^^^^^^^^
  File "C:\Users\K K\anaconda3\envs\gameplan\Lib\site-packages\jose\jws.py", line 191, in _load
    raise JWSError("Invalid header string: %s" % e)
jose.exceptions.JWSError: Invalid header string: 'utf-8' codec can't decode byte 0x88 in position 6: invalid start byte

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Gameplan\GamePlan-Sports-Team-Match-Scheduling-API\app\auth.py", line 14, in get_current_user
    payload = jwt.decode(token, SECRET_KEY, algorithms=[ALGORITHM])
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\K K\anaconda3\envs\gameplan\Lib\site-packages\jose\jwt.py", line 161, in decode
    raise JWTError(e)
jose.exceptions.JWTError: Invalid header string: 'utf-8' codec can't decode byte 0x88 in position 6: invalid start byte
2025-09-04 17:49:51,917 - WARNING - Team creation failed: Royel Challengers already exists
